config/community_images.json

```diff
@@ -7,4 +7,14 @@
     "image": "cumulusprod/pegasus-terra:1.0",
     "requiresSpark": false,
     "isCommunity": true
+},
+{
+    "id": "RStudio",
+    "label": "RStudio (R 4.0.0, Bioconductor 3.11.1, Python 2.7.17)",
+    "version": "0.0.6",
+    "updated": "2020-07-01",
+    "packages": "https://storage.googleapis.com/terra-docker-image-documentation/placeholder.json",
```

Comment:
So we should ask the Bioconductor team to generate a list of packages, correct? (I think we talked about this in person)

Reply:
Yeah, I was planning on letting them know once this is released but now that I think about it, I should probably reach out sooner so that their json file is in place once the release happens